web/assets/js/app.js

```diff
@@ -1,83 +1,40 @@
-jQuery(document).ready(function(){
-    const $ = jQuery;
-    // credits to http://codyhouse.co/gem/css-jquery-image-comparison-slider/
-    const drags = function (dragElement, resizeElement, container) {
-        dragElement.on('mousedown vmousedown', function (e) {
-            dragElement.addClass('draggable');
-            resizeElement.addClass('resizable');
-            const dragWidth = dragElement.outerWidth(),
-                xPosition = dragElement.offset().left + dragWidth - e.pageX,
-                containerOffset = container.offset().left,
-                containerWidth = container.outerWidth(),
-                minLeft = containerOffset + 10,
-                maxLeft = containerOffset + containerWidth - dragWidth - 10;
-            dragElement.parents().on('mousemove vmousemove', function (e) {
-                let leftValue = e.pageX + xPosition - dragWidth;
-                if(leftValue < minLeft ) {
-                    leftValue = minLeft;
-                } else if ( leftValue > maxLeft) {
-                    leftValue = maxLeft;
-                }
-                const widthValue = (leftValue + dragWidth/2 - containerOffset)*100/containerWidth+'%';
-                $('.draggable').css('left', widthValue).on('mouseup vmouseup', function () {
-                    $(this).removeClass('draggable');
-                    resizeElement.removeClass('resizable');
-                });
-                $('.resizable').css('width', widthValue);
-            }).on('mouseup vmouseup', function (e){
-                dragElement.removeClass('draggable');
-                resizeElement.removeClass('resizable');
-            });
-            e.preventDefault();
-        }).on('mouseup vmouseup', function (e) {
-            dragElement.removeClass('draggable');
-            resizeElement.removeClass('resizable');
-        });
-    };
-
-    $('.cd-resize-img img').width($('.cd-image-container img').css('width'));
-
-    drags($('.cd-handle'), $('.cd-resize-img'), $('.cd-image-container'));
-    $('.cd-image-container').addClass('is-visible');
-
-    $(window).on('resize', function () {
-        $('.cd-resize-img img').width($('.cd-image-container img').css('width'));
-    });
-
-    const locationHash = document.location.hash;
+jQuery(document).ready(function() {
```

Comment:
Why not `(function() { ... })();`

Reply:
This file will be completely change so I decided to focus only on Google Analytics part.